backend/src/main/java/ca/mcgill/purposeful/controller/AppUserController.java

```diff
@@ -81,4 +90,123 @@ public ResponseEntity<AppUserDto> registerModerator(@RequestBody AppUserDto appU
     return ResponseEntity.status(HttpStatus.CREATED).body(registeredUser);
   }
 
+  /**
+   * PUT method to update the names of a regular user
+   * @param appUserDto - the user to modify the names
+   * @return the modified user
+   * 
+   * @author Enzo Benoit-Jeannin
+   */
+    @PutMapping(value = {"/regular",
+      "/regular/"}, consumes = "application/json", produces = "application/json")
+    @PreAuthorize("hasAnyAuthority('Owner', 'Moderator', 'User')")
```

Comment:
This endpoint should be restricted to the concerned user only so that other users cannot modify the information. Is that authentication done?

Reply:
I added the restriction in my last pull request (step definitions)